src/base/ClpMap.h

```diff
@@ -93,6 +93,14 @@ class ClpMap
     /// Remove the corresponding entry (if any)
     void del(const Key &);
 
+    /// Merge entries from another ClpMap.
+    ///  * Entries imported from the other list will retain their LRU order
+    ///    and be placed before existing entries.
+    ///  * Entries which have expired will be ignored.
+    ///  * When an key collision occurs the existing entry will be replaced
+    ///    and the new entry given the largest of the two TTL values.
+    void merge(const ClpMap<Key, Value, MemoryUsedBy> &other);
```

Comment:
Please rename this method to importFresh(). There are many reasonable ways to merge two maps, and we should give this specific merging algorithm a more specific name. While renaming, please avoid the term "list" when mentioning this new method argument -- ClpMap is an associative container (i.e. a map) rather than a list.